testbed/tests/test_window.py

```diff
@@ -26,7 +26,8 @@ async def second_window(second_window_kwargs):
 
 
 @pytest.fixture
-async def second_window_probe(app, second_window):
+async def second_window_probe(app_probe, second_window):
+    app = app_probe.app
```

Comment:
There's no cost to including both fixtures, and then ignoring one - in fact, its a common way for pytest to do test setup (e.g., a fixture that creates a directory structure).
```suggestion
async def second_window_probe(app, app_probe, second_window):
```